tests/test_consumer/test_extractors/test_extractors_column_lineage.py

```diff
@@ -0,0 +1,354 @@
+import pytest
+from uuid6 import UUID
+
+from data_rentgen.consumer.extractors import extract_batch
+from data_rentgen.consumer.extractors.column_lineage import (
+    extract_column_lineage,
+    extract_dataset_column_relation_type,
+)
+from data_rentgen.consumer.openlineage.dataset_facets import (
+    OpenLineageColumnLineageDatasetFacet,
+    OpenLineageColumnLineageDatasetFacetField,
+    OpenLineageColumnLineageDatasetFacetFieldRef,
+    OpenLineageColumnLineageDatasetFacetFieldTransformation,
+)
+from data_rentgen.dto import (
+    ColumnLineageDTO,
+    DatasetColumnRelationDTO,
+    DatasetColumnRelationTypeDTO,
+    DatasetDTO,
+)
+from tests.test_consumer.test_extractors.fixtures.column_lineage_facets import (
+    get_run_event_with_column_lineage,
+)
+from tests.test_consumer.test_extractors.fixtures.extracted_dto import get_operation_dto
+
+
+def test_extractors_extract_dataset_column_relation_type_new_type():
+    transformation = OpenLineageColumnLineageDatasetFacetFieldTransformation(
+        type="SOME_NEW_TYPE",
+    )
+    dataset_column_relation_type = extract_dataset_column_relation_type(transformation)
+    assert dataset_column_relation_type == DatasetColumnRelationTypeDTO.UNKNOWN
+
+
+@pytest.mark.parametrize("type", ("INDIRECT", "DIRECT"))
+def test_extractors_extract_dataset_column_relation_type_no_subtype(type):
+    transformation = OpenLineageColumnLineageDatasetFacetFieldTransformation(
+        type=type,
+    )
+    dataset_column_relation_type = extract_dataset_column_relation_type(transformation)
+    assert dataset_column_relation_type == DatasetColumnRelationTypeDTO.UNKNOWN
+
+
+@pytest.mark.parametrize(
+    ("subtype", "expected_type"),
+    (
+        ("TRANSFORMATION", DatasetColumnRelationTypeDTO.TRANSFORMATION_MASKING),
+        ("AGGREGATION", DatasetColumnRelationTypeDTO.AGGREGATION_MASKING),
+        ("NEW_MASKING_SUBTYPE", DatasetColumnRelationTypeDTO.UNKNOWN),
```

Comment:
Let's replace this item with some INDIRECT type. We already tested above that unknown transformation types are converted to UNKNOWN.

Reply:
Some *existing* indirect type